### PR TITLE
Network 25372: Global Secure Access (GSA) client is deployed on all managed endpoints

### PR DESCRIPTION
**Pull request by ashwinikarke**

### CODE REVIEW

File: src/powershell/tests/Test-Assessment.25372.ps1 [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    $customStatus = $null
+
+    # Edge case: GSA devices > Intune devices (data inconsistency; GSA has more devices than Intune)
+    if ($totalGsaDevices -gt $totalManagedDevices -and $totalManagedDevices -gt 0) {
+        $customStatus = 'Investigate'
+        $deploymentPercentage = 'N/A'
+        $gap = 'N/A'
+        $testResultMarkdown = "⚠️ Global Secure Access device count exceeds the Intune-managed device count. This indicates stale GSA device records, devices removed from Intune management, or data synchronization issues between systems. Review both data sources to reconcile counts.`n`n%TestResult%"
+    }
+    # Edge case: No devices at all (both = 0) - Fail per spec
```
[the remark this answers is not in the document]
Comment (alexandair) [reply src/powershell/tests/Test-Assessment.25372.ps1]: @ashwinikarke I'm confused. What happened to "Investigate (No Data)" option in the spec? Why is it removed?
It still exists in code and wrongly fails instead of investigate.

**Comment by ashwinikarke** [comment]
Created New PR https://github.com/microsoft/zerotrustassessment/pull/844
